tests/v2/validation/rbac/rbacuserroles/cluster_role_test.go

```diff
@@ -0,0 +1,141 @@
+	require.NoError(rb.T(), err)
+
+	rb.client = client
+
+	log.Info("Getting cluster name from the config file and append cluster details in rb")
+	clusterName := client.RancherConfig.ClusterName
+	require.NotEmptyf(rb.T(), clusterName, "Cluster name to install should be set")
+	clusterID, err := clusters.GetClusterIDByName(rb.client, clusterName)
+	require.NoError(rb.T(), err, "Error getting cluster ID")
+	rb.cluster, err = rb.client.Management.Cluster.ByID(clusterID)
+	require.NoError(rb.T(), err)
+}
+
+func (rb *RBACAdditionalTestSuite) TestRbacClusterOwnerChecks() {
+	subSession := rb.session.NewSession()
+	defer subSession.Cleanup()
+
+	clusterAdmin, standardUserClient, err := rbac.SetupUser(rb.client, rbac.StandardUser)
+	require.NoError(rb.T(), err)
+
+	additionalUser, additionalUserClient, err := rbac.SetupUser(rb.client, rbac.StandardUser)
+	require.NoError(rb.T(), err)
+
+	rb.T().Logf("Adding user as " + rbac.RoleOwner + " to the downstream cluster.")
+	err = users.AddClusterRoleToUser(rb.client, rb.cluster, clusterAdmin, rbac.RoleOwner, nil)
+	require.NoError(rb.T(), err)
+	standardUserClient, err = standardUserClient.ReLogin()
+	require.NoError(rb.T(), err)
+
+	rb.T().Logf("Validating if " + rbac.RoleOwner + " can add another standard user as a project owner")
+	clusterOwnerProject, err := rbac.CreateProject(standardUserClient, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	err = users.AddProjectMember(standardUserClient, clusterOwnerProject, additionalUser, rbac.RoleProjectOwner, nil)
+	require.NoError(rb.T(), err)
+
+	projectListClusterAdditionalUser, err := projects.ListProject(additionalUserClient, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	assert.Equal(rb.T(), 1, len(projectListClusterAdditionalUser.Data))
+	assert.Equal(rb.T(), clusterOwnerProject.Name, projectListClusterAdditionalUser.Data[0].Name)
+
+	err = users.RemoveProjectMember(standardUserClient, additionalUser)
+	require.NoError(rb.T(), err)
+
+}
+
+func (rb *RBACAdditionalTestSuite) TestRbacClusterOwnerAdds() {
+	subSession := rb.session.NewSession()
+	defer subSession.Cleanup()
+
+	_, standardUserClient, err := rbac.SetupUser(rb.client, rbac.StandardUser)
+	require.NoError(rb.T(), err)
+
+	additionalUser, additionalUserClient, err := rbac.SetupUser(rb.client, rbac.StandardUser)
+	require.NoError(rb.T(), err)
+
+	rb.T().Logf("Validating if " + rbac.RoleOwner + " can add another standard user as a cluster owner")
+	errUserRole := users.AddClusterRoleToUser(standardUserClient, rb.cluster, additionalUser, rbac.RoleOwner, nil)
+	require.NoError(rb.T(), errUserRole)
+	additionalUserClient, err = additionalUserClient.ReLogin()
+	require.NoError(rb.T(), err)
+
+	clusterList, err := additionalUserClient.Steve.SteveType(clusters.ProvisioningSteveResourceType).ListAll(nil)
+	require.NoError(rb.T(), err)
+	assert.Equal(rb.T(), 1, len(clusterList.Data))
+
+	err = users.RemoveClusterRoleFromUser(standardUserClient, additionalUser)
+	require.NoError(rb.T(), err)
+
```

Comment:
Assertion is missing?

Reply:
as this is a refactor, I haven't changed the existing logic for any of these tests but updated the formatting but sure I can add an assertion here.